src/App.tsx

```diff
@@ -47,19 +46,14 @@ import { AddLiquidity } from 'pages/earn/pools/AddLiquidity';
 export const App = () => {
   const dispatch = useDispatch();
   const { chainId, account } = useWeb3React();
-  const [loading, setLoading] = useState(true);
+  useAutoConnect();
   const unsupportedNetwork = isUnsupportedNetwork(chainId);
-  const triedAutoLogin = useAutoConnect();
   const [isSidebarMinimized, setIsSidebarMinimized] = useState(false);
   const [isSidebarOpen, setIsSidebarOpen] = useState(false);
   const notifications = useAppSelector<Notification[]>(
     (state) => state.notification.notifications
   );
 
-  useEffect(() => {
-    if (chainId || triedAutoLogin || !getAutoLoginLS()) setLoading(false);
```

Comment:
I'm worried about gnosis support. Also, you no longer need to import `getAutoLoginLS`